Gravity-tube/src/Components/Modal/Modal.css

```diff
@@ -0,0 +1,74 @@
+/* playlist */
+
+.playlist-conatainer{
+  position: fixed;
+  height: 100vh;
+  width: 100%;
+  z-index: 15;
+  top: 0%; 
+  left: -1%;
+  background-color: #000000cc;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  }
+  .playlist-box{
```

Comment:
- suggestion: give space between two classes, and also between curly brackets and class name.